notebooks/MAST/dwarf_galaxy_search/dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    condition = "nDetections.gte=1" #this condition allows us to only get data where at least 1 observation has been made
</pre><p>I think you didn't apply more filters I suggested before (I don't mean about the Held paper criteria as an option). I think I left those comments somewhere below about CMD. Please implement those filters (remove duplicates, etc..)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>sorry i did miss those comments before, i just didnt see them until now</p>